include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -23,27 +23,14 @@
 extern "C" {
 #endif
 
-/* AT Commands */
-enum bt_hfp_hf_at_cmd {
-	BT_HFP_HF_ATA,
-	BT_HFP_HF_AT_CHUP,
-};
+/* HFP CODEC IDs */
+#define BT_HFP_HF_CODEC_CVSD    0x01
+#define BT_HFP_HF_CODEC_MSBC    0x02
+#define BT_HFP_HF_CODEC_LC3_SWB 0x03
```

Comment:
Should we consider adding a public common header file for these defines on AG and HFP? There are many similar definitions in HF and AG.

Reply:
Currently, HF and AG have their own marcos.

For HFP HF,
`/* HFP CODEC IDs */
#define BT_HFP_HF_CODEC_CVSD    0x01
#define BT_HFP_HF_CODEC_MSBC    0x02
#define BT_HFP_HF_CODEC_LC3_SWB 0x03`

For HFP AG,
`/* HFP CODEC */
#define BT_HFP_AG_CODEC_CVSD    0x01
#define BT_HFP_AG_CODEC_MSBC    0x02
#define BT_HFP_AG_CODEC_LC3_SWB 0x03`

I don't see any benefit in adding a header file for these three macros.

Reply:
Are these the only only shared macros, or are there more? Are these independently specified (separately for each role) in the HFP spec? If there are more than just these three, and if they are not specified as being independent for each role in the HFP spec, then it seems quite justified to me to move them to a common `hfp.h` file.

Reply:
> Are these the only only shared macros, or are there more? 

Currently, only these macros. Others are used internally.

> Are these independently specified (separately for each role) in the HFP spec? 

They are same regardless role.

> If there are more than just these three, and if they are not specified as being independent for each role in the HFP spec, then it seems quite justified to me to move them to a common `hfp.h` file.

I don't think it's necessary to add a file for these three macros. If there are other needs in the future, I think you can consider putting it in a public file.